packages/codegen-ui/example-schemas/forms/post-datastore-relationship-multiple.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "BookCreateForm",
+  "formActionType": "create",
+  "dataType": {
+    "dataSourceType": "DataStore",
+    "dataTypeName": "Book"
+  },
+  "fields": {
+    "primaryAuthor": {
+      "inputType": {
+        "type": "Autocomplete",
```

Comment:
nit: the two form schemas added are titled "post-" but use "Book" schemas

Reply:
updated